seed/models/building_file.py

```diff
@@ -123,17 +124,20 @@ def process(self, organization_id, cycle, property_state=None, property_id=None)
                     is_extra_data=True,
                 )
 
-        if property_state and property_id:
-            property_state.promote(cycle, property_id)
-        elif not property_state and not property_id:
-            # create a new property_state for the objects
-            property_state = PropertyState.objects.create(**create_data)
-            property_state.extra_data = extra_data
-            property_state.promote(cycle)
-            property_state.save()
-        else:
-            # invalid arguments, must pass both or neither
-            return False, None, "Invalid arguments passed to BuildingFile.process()"
+        # always create the new object, then decide if we need to merge it.
+        # create a new property_state for the object and promote to a new property_view
+        property_state = PropertyState.objects.create(**create_data)
```

Comment:
Interesting.  I thought we were trying to avoid creating a new property state.

Reply:
we were. I changed my mind because i wanted to leverage the merge functionality instead of doing 'magic' on the propertystate itself. The merge requires 2 states to be passed.